### PR TITLE
[DONE] Lars filter user input

### PR DESCRIPTION
**Pull request by larsclaussen**

### CODE REVIEW

**Comment by larsclaussen** [comment]
[[plugin] check number of connections for connected calc points [1]](https://trello.com/c/J0R1kB94/965-plugin-check-number-of-connections-for-connected-calc-points-1)

File: views/predict_calc_points_dialog.py [hunk 1]
```diff
@@ -140,3 +140,119 @@ def retranslateUi(self):
...
+        """Accept and run the Command.run_it method."""
+
+
+        db_key = self.database_combo.currentText()
+        db_entry = self.databases[db_key]
+
+        _db_type = db_entry['db_type']
+        _db_settings = db_entry['db_settings']
+
+        if _db_type == 'sqlite':
```
Comment (jackieleng) [inline views/predict_calc_points_dialog.py]: Maybe candidate for enhancement?

Comment (larsclaussen) [reply views/predict_calc_points_dialog.py]: Sure. But could you be a bit more specific about what you think?

File: commands/toolbox_tools/stap 3 - Schematisatie bewerken/add_connected_points.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            connected_pnt = dict(
+                zip(
+                    self.fnames_connected_pnt, feat.attributes()
+                )
+            )
+            calculation_pnt_id = connected_pnt['calculation_pnt_id']
+            calc_pnt_request = QgsFeatureRequest().setFilterExpression(
+                u'"id" = {}'.format(calculation_pnt_id))
+            calc_pnt_feat = self.calc_pnt_lyr.getFeatures(
+                calc_pnt_request).next()
```
Comment (jackieleng) [inline commands/toolbox_tools/stap 3 - Schematisatie bewerken/add_connected_points.py]: StopIteration is raised here if you add a point in the incorrect way.

**Pull request merged**